### ISSUE TITLE
Patch destroyed by the 13th Dec. Update

### ISSUE THREAD

**Issue by DanielCruz182** [Maxx53/NoCableLauncher#10]
The game was updated today and the patch stopped working, in the new input settings it reads the interface as a microphone but no sound and in the "real tone cable" option doesn't recognize anything.
 Thank you for the great work, I would be very thankfull if you could fix the issue.

**Comment by Maxx53** [comment]
Yep, we need new offcets. I guess It could be found in memory by pid&vid of original realtone cable.

**Comment by Maxx53** [comment]
Here we go...
VID offcet 012DE6EC
PID offcet 012DE6F4

Can't check it right now.
There's some guess. You must turn off all other input devices except one connected to guitar.

**Comment by unkier** [comment]
Yep, it works for me. Thanks man !

**Comment by DanielCruz182** [comment]
Thank you, it works like a charm!

**Comment by ss23** [comment]
How much effort is it to include an offset finder within the application to make new patches easier to work around in future, @Maxx53? I can open a new issue if you'd like.

**Comment by Maxx53** [comment]
@ss23, I believe It's possible. First, I need some different versions of "Rocksmith2014.exe" to test search as universal.

**Comment by joshfoss** [comment]
Hey guys, any idea if this will work with my guitar connected to my steinberg usb audio interface? I typically have my guitar connected to it for recording purposes.

I dont own rocksmith yet, but wanted to check before I buy.

**Comment by Maxx53** [comment]
@joshfoss Can't say for sure. Check it on pirated version.

**Comment by Airtrooper719** [comment]
Thanks for raising this and also thanks for the solution much appreciated :)
